index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="https://kit.fontawesome.com/0bd1156d30.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="Main-Page">
+     <nav class="nav">
+<div class="logo">AFIAMOAH</div>
+
+<div class="nav-links-menu"><p class="side-logo"><i class="fa-solid fa-bars"></i></p></div>
+
+     </nav>
+     <section class="section">
+<h1>Hey There. I’m Afiamoah</h1>
+<h2>I’am a Software Developer</h2>
+<p> class="content">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.</p>
+    <div class="imgcontainer">
+        <div class="images"><img src="linkedin.png" alt=""></div>
+        <div class="images"><img src="Enabled.png" alt=""></div>
+        <div class="images"><img src="twitter.png" alt=""></div>
+        <div class="images"><img  id="Medium" src="Medium.png" alt=""></div>
+        
+    </div>
```

Comment:
- [ ] Since the set of social links is structurally a list of items, kindly replace the divs with `ul` and `li`'s like below to improve the semantic structure of the website. Remember that div's hold no semantic meaning so you should minimize its use to only containers and wrappers.
```
  <ul class="imgcontainer">
        <li class="images"><img src="linkedin.png" alt=""></li>
        <li class="images"><img src="Enabled.png" alt=""></li>
        <li class="images"><img src="twitter.png" alt=""></li>
        <li class="images"><img  id="Medium" src="Medium.png" alt=""></li>
    </ul>
```

- [ ] Please add all of the required social links, you might've missed GitHub in your design. Additionally, the social links should not have a border as you can see in the comparison below.
> Figma
![image](https://user-images.githubusercontent.com/90556221/176878239-e8a43a9f-531c-4f55-a4f8-ad32074ef567.png)

> Yours
![image](https://user-images.githubusercontent.com/90556221/176878288-baa64d37-d7bb-4165-b6b1-029947fa9623.png)

Reply:
Missing icon(github icon) added succesfully